server/src/main/java/org/opensearch/index/shard/InternalIndexingStats.java

```diff
@@ -43,9 +43,9 @@
 
 /**
  * Internal class that maintains relevant indexing statistics / metrics.
- * @see IndexShard
  *
  * @opensearch.internal
+ * @see IndexShard
```

Comment:
nit: I guess `@see IndexShard` was rightly placed after the description on L46 as it seemed in continuation to the comment.